### PR TITLE
Properly resolving target file location for block macros

### LINKED ISSUE #446: Invalid resolution of files in block macro for non-antora setup
## Setup
```
.
└── sample
    ├── _assets
    │   └── test.puml
    └── test.adoc
```
With the following content in `test.adoc`:
```
= Test document

Take a look at plantuml feature:

plantuml::_assets/test.puml[]
```

## Problem

I'm converting file with the following command: `asciidoctor -r asciidoctor-kroki -D out sample/test.adoc`

**Expected behavior**: `_assets/test.puml` file is propertly resolved _relative to_ `test.adoc`, whenever I'm located in parent dir or in `sample` 

**Actual behavior**: I got error:
> sciidoctor: ERROR: test.adoc: line 5: Failed to read plantuml file: _assets/test.puml. No such file or directory @ rb_sysopen - _assets/test.puml.

File is only converted correctly when I place `_assets` dir, _relative to_ location of workdir where I execute command.

## Comparision with asciidoctor-diagram

`asciidoctor-diagram` module properly resolve file, I dont' get error with the following command: `asciidoctor -r asciidoctor-diagram -D out sample/test.adoc`

## Possible issue

kroki extension uses raw `target` without resolution: 
* https://github.com/asciidoctor/asciidoctor-kroki/blob/master/ruby/lib/asciidoctor/extensions/asciidoctor_kroki/extension.rb#L83
* https://github.com/asciidoctor/asciidoctor-kroki/blob/master/ruby/lib/asciidoctor/extensions/asciidoctor_kroki/extension.rb#L65
* https://github.com/asciidoctor/asciidoctor-kroki/blob/master/ruby/lib/asciidoctor/extensions/asciidoctor_kroki/extension.rb#L96

To compare, `asciidoctor-diagram` resolves target with `normalize_system_path` method: https://github.com/asciidoctor/asciidoctor-diagram/blob/9f1f9352bff950df0eae7790a85b5cd85c99c2f0/lib/asciidoctor-diagram/diagram_processor.rb#L396

### PR DESCRIPTION
**Pull request by zeldigas**
Fixes #446

### CODE REVIEW

**Comment by ggrossetie** [comment]
Thanks, would it be possible to revert the version bump and add a test with an example that wasn't working?

**Comment by ggrossetie** [comment]
@zeldigas I rebased your branch on master because the CI was running the JavaScript build instead of the Ruby build 😒

**Comment by zeldigas** [comment]
> Thanks, would it be possible to revert the version bump and add a test with an example that wasn't working?

@ggrossetie I'll revert version bump, and will try to add test for issue. Will let you know if I have problem with the latter

**Comment by ggrossetie** [comment]
@zeldigas let me know if you need or guidance on adding a test.

File: ruby/spec/asciidoctor_kroki_block_macro_spec.rb [hunk 1]
```diff
@@ -92,7 +104,7 @@ def resolve_target_path(target)
     it 'should display unresolved block macro message when the target cannot be resolved' do
       # noinspection RubyClassModuleNamingConvention
       class UnresolvedTargetKrokiBlockMacroProcessor < ::AsciidoctorExtensions::KrokiBlockMacroProcessor
-        def resolve_target_path(_target)
+        def resolve_target_path(_target, _parent)
```
Comment (ggrossetie) [inline ruby/spec/asciidoctor_kroki_block_macro_spec.rb:107]: We are not using the parameters but I guess it should be `_parent` (first parameter) then `_target`
```suggestion
        def resolve_target_path(_parent, _target)
```

Comment (zeldigas) [reply ruby/spec/asciidoctor_kroki_block_macro_spec.rb:107]: Will fix

**Comment by zeldigas** [comment]
> @zeldigas let me know if you need or guidance on adding a test.

Thank you, I'll prepare change, but most likely tomorrow or on weekend. Workdays are usually stuffed with work activities.

**Comment by zeldigas** [comment]
@ggrossetie  test added

**Comment by ggrossetie** [comment]
Looks good, thanks!

**Pull request merged**